src/views/PageSearchPortal.vue

```diff
@@ -85,28 +80,26 @@ export default {
     created() {
         this.getParkingRequests();
         if (this.SOLatLngInput) {
-            this.getInterestedVO();
+            this.getInterestedVO(this.SOLatLngInput);
         }
     },
     methods: {
         ...mapActions('searchPortal', [
             'updateActiveTab',
             'updateSOLatLngInput',
         ]),
-        async getParkingRequests() {
+        async getParkingRequests(voMobile = '') {
             this.isLoading = true;
             let parkingRequestURL = '/internal/parking-requests';
-            if (this.VOMobile != '') {
-                parkingRequestURL =
-                    parkingRequestURL +
-                    `?mobile=${this.VOMobile.replace(/\s+/g, '')}`;
+            if (voMobile != '') {
+                parkingRequestURL = parkingRequestURL + `?mobile=${voMobile.replace(/\s+/g, '')}`;
             }
             this.parkingRequests = await mayaClient.get(parkingRequestURL);
```

Comment:
Please wrap the API call inside try-catch

Reply:
mayaClient is handling these
https://github.com/arrogant-ant/parkspot/blob/master/src/services/api.js#L102